features/junit_format.feature

```diff
@@ -538,6 +538,7 @@
       """
     And the file "junit/default.xml" should be a valid document according to "junit.xsd"
 
+  @php-version @php5.3 @php5.4 @php5.5 @php5.6
```

Comment:
PHP 5.6 already runs the scenario tagged as ``@php5.4``, due to the way our Travis config works. It is meant to be a min version (and the code used in the Travis setup is currently broken for PHP 7+ when generating the list btw)